test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs

```diff
@@ -257,7 +257,7 @@ public class ScriptFileGetLinesTests
     {
         private ScriptFile scriptFile;
 
-        private const string TestString = "Line One\r\nLine Two\r\nLine Three\r\nLine Four\r\nLine Five";
+        private const string TestString = "Line One\r\nLine Two\r\nLine Three\r\nLine Four\r\nLine Five\r\n";
```

Comment:
Was this meant to test content with no trailing EOL?  If not, then shouldn't this field be used on line 270 instead of the literal string there?

Reply:
Ok, I've got both tested now. Take a look if you can